.sync.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+mock_with: ':rspec'
+
+.gitlab-ci.yml:
+  bundler_args: |
+    --without system_tests --path vendor/bundle
+    cache:
+      key: "$CI_JOB_STAGE-$CI_COMMIT_REF_NAME"
+      paths:
+        - vendor/bundle
+.travis.yml:
```

Comment:
we need to update these settings if you will use the newest pdk template version.
They updated how the deployment works. I think this causes two deployment jobs now so we need to remove some of our old way of doing that.